src/unix/kqueue.c

```diff
@@ -284,6 +284,15 @@ void uv__io_poll(uv_loop_t* loop, int timeout) {
     loop->watchers[loop->nwatchers + 1] = (void*) (uintptr_t) nfds;
     for (i = 0; i < nfds; i++) {
       ev = events + i;
+#if defined(__APPLE__)
```

Comment:
I think we should be able to use this code for all platforms which have kqueue, so let's remove this ifdef and use this everywhere that we can instead

Reply:
How should we detect whether kqueue is available in `src/unix/process.c`? from the build system it looks like the condition is `if(APPLE OR CMAKE_SYSTEM_NAME MATCHES "DragonFly|FreeBSD|NetBSD|OpenBSD")`, so should this be replaced with `#if defined(__APPLE__) || defined(__DragonFly__) || defined(__FreeBSD__) || defined(__NetBSD__) || defined(__OpenBSD__)`?

Reply:
Cool, this is done then.